src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -1257,6 +1306,15 @@ void DoorLockServer::setYearDayScheduleCommandHandler(chip::app::CommandHandler
                     "[endpointId=%d,yearDayIndex=%d,userIndex=%d,localStartTime=%" PRIu32 ",endTime=%" PRIu32 "]",
                     endpointId, yearDayIndex, userIndex, localStartTime, localEndTime);
 
+    // Update LastModifiedFabricIndex of user
+    if (!emberAfPluginDoorLockSetUser(endpointId, userIndex, user.createdBy, fabricIdx, user.userName, user.userUniqueId,
```

Comment:
Seems like this should be a helper function taking `EmberAfPluginDoorLockUserInfo` and `FabricIndex`?